app/plugins.js

```diff
@@ -228,46 +223,47 @@ function toDependencies(plugins) {
 }
 
 function install(fn) {
-  const {shell: cfgShell, npmRegistry} = exports.getDecoratedConfig();
+  const yarnPath = resolve(__dirname, '..', 'bin', 'yarn-standalone.js');
```

Comment:
`yarnPath` is not used elsewhere then inside `yarn` function. Better to move it inside function scope.